medicines/doc-index-updater/src/create_manager/metadata.rs

```diff
@@ -0,0 +1,147 @@
+use lazy_static;
+use regex::Regex;
+use std::str;
+
+pub fn sanitize(s: &str) -> String {
+    s.replace(|c: char| !c.is_ascii(), "")
```

Comment:
ASCII seems overly restrictive, what's the deciding factor for which characters are acceptable?